lib/service/Controller/Swap/AXT.dart

```diff
@@ -8,6 +8,13 @@ class AXT extends Token with ERC20Behavior {
   static final String mumbaiAddress =
       "0x76d9a6e4cdefc840a47069b71824ad8ff4819e85";
```

Comment:
Would it useful to do something like 
static final String txAddress = (kReleaseMode) ? polygonAddress : mumbaiAddress